analysis/parser/RawTextParser/src/main/kotlin/de/maibornwolff/codecharta/parser/rawtextparser/ParserDialog.kt

```diff
@@ -34,8 +34,9 @@ class ParserDialog {
                 hint = "metric1, metric2, metric3 (leave empty for all metrics)"
             )
 
-            val tabWidth: BigDecimal =
-                KInquirer.promptInputNumber(message = "What is the tab width used (estimated if not provided)?")
+            val tabWidth: String =
+                KInquirer.promptInput(message = "What is the tab width used (estimated if unknown)?", default = "unknown", hint = "unknown")
```

Comment:
```suggestion
                KInquirer.promptInput(message = "What is the tab width used (estimated if unknown)?", default = "unknown", hint = "4")
```

Shall we mention that the tabWidth is counted in spaces / tabs? I think it is not clear, what a tabWidth 4 actually means.

Reply:
Do I understand it right, that the tabWidth is needed only when spaces are used as  \t.
I ask this question to better understand how to formulate the question, so that it is more understandable.

Reply:
Maybe we could ask something like "If spaces are used for indentations, how many of them represent one indentation level aka tab width?"